app/src/main/java/com/sun_asterisk/foodies/screen/main/layout_adapter/ViewPagerAdapter.kt

```diff
@@ -0,0 +1,28 @@
+package com.sun_asterisk.foodies.screen.main.layout_adapter
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
+
+class ViewPagerAdapter(fm: FragmentManager) :
+    FragmentPagerAdapter(fm, FragmentPagerAdapter.BEHAVIOR_RESUME_ONLY_CURRENT_FRAGMENT) {
+    var mFragmentList: MutableList<Fragment> = ArrayList()
```

Comment:
```suggestion
    private val fragmentList = mutableListOf<Fragment>()
```

Reply:
@ThanhDeveloper Check this comment again!

Reply:
Em fix rồi ạ